sensor/marker_radar_lidar_calibrator/CMakeLists.txt

```diff
@@ -15,12 +17,14 @@ ament_export_include_directories(
 
 ament_auto_add_executable(marker_radar_lidar_calibrator
   src/marker_radar_lidar_calibrator.cpp
+  src/transformation_estimator.cpp
   src/track.cpp
   src/main.cpp
 )
 
 target_link_libraries(marker_radar_lidar_calibrator
   ${OpenCV_LIBS}
+  ${CERES_LIBRARIES}
```

Comment:
Could you test if we can avoid manual `target_link_libraries` since we are using `ament_auto`?

Reply:
This happens because you did not put ceres in the packages file

`<depend>libceres-dev</depend>`

Reply:
Same, even adding the https://github.com/tier4/CalibrationTools/pull/161/commits/3cf1228e2cf427b16c2ecf8725528cdf1ce57cdf, still couldn't remove the find_package and link_library :crying_cat_face: